server/room-game.ts

```diff
@@ -18,7 +18,7 @@
 // globally Rooms.RoomGamePlayer
 export class RoomGamePlayer {
 	readonly num: number;
-	readonly game: RoomGame;
+	readonly roomid: RoomID;
```

Comment:
Why was this changed?

Reply:
I changed this so that I could use `RoomGamePlayer`s to track Mastermind players without technically having a `RoomGame` in progress.

Reply:
This is no longer necessary.